users/tests/test_api_user.py

```diff
@@ -0,0 +1,222 @@
+) -> None:
+    client, _, _ = authenticated_client
+    response = client.get(api_user_settings_url)
+    assert response.status_code == 200
+    assert response.data == {
+        "id": user.usersettings.id,
+        "content_lang": user.usersettings.content_lang,
+        "location": user.usersettings.location,
+        "is_beta": user.usersettings.is_beta,
+        "revert_to_old_reddit": user.usersettings.revert_to_old_reddit,
+        "is_over_18": user.usersettings.is_over_18,
+        "user": user.id,
+    }
+
+
+@pytest.mark.django_db()
+def test_patch_api_user_settings_valid_data_response_success(
+    authenticated_client: tuple[APIClient, str, str], api_user_settings_url: str, user: User
+) -> None:
+    client, _, _ = authenticated_client
+    data = {"is_beta": not user.usersettings.is_beta}
+    response = client.patch(api_user_settings_url, data=data)
+    assert response.status_code == 200
+    assert response.data["is_beta"]
+    assert response.data == {
+        "id": user.usersettings.id,
+        "content_lang": user.usersettings.content_lang,
+        "location": user.usersettings.location,
+        "is_beta": not user.usersettings.is_beta,
+        "revert_to_old_reddit": user.usersettings.revert_to_old_reddit,
+        "is_over_18": user.usersettings.is_over_18,
+        "user": user.id,
+    }
+
+
+@pytest.mark.django_db()
+def test_patch_api_user_settings_invalid_data_response_failed(
+    authenticated_client: tuple[APIClient, str, str], api_user_settings_url: str, user: User
+) -> None:
+    client, _, _ = authenticated_client
+    data = {"is_beta": "TEST"}
+    response = client.patch(api_user_settings_url, data=data)
+    assert response.status_code == 400
+
+
+@pytest.mark.django_db()
+def test_api_user_settings_unauthenticated_user_response_failed(
+    client: Client, user_credentials: tuple[APIClient, str, str], api_user_settings_url: str
+) -> None:
+    response = client.get(api_user_settings_url)
+    assert response.status_code == 401
+    assert isinstance(response.data["detail"], ErrorDetail)
+
+
+@pytest.mark.django_db()
+def test_patch_api_user_settings_is_beta_response_success(
+    authenticated_client: tuple[APIClient, str, str], api_user_settings_url: str, user: User
+) -> None:
+    client, _, _ = authenticated_client
+    data = {"is_beta": not user.usersettings.is_beta}
+    response = client.patch(api_user_settings_url, data=data)
+    assert response.status_code == 200
+    assert response.data["is_beta"] == (not user.usersettings.is_beta)
+
+
+@pytest.mark.django_db()
+def test_patch_api_user_settings_content_lang_response_success(
+    authenticated_client: tuple[APIClient, str, str], api_user_settings_url: str, user: User
+) -> None:
+    client, _, _ = authenticated_client
+    data = {"content_lang": "es"}
+    response = client.patch(api_user_settings_url, data=data)
+    assert response.status_code == 200
+    assert response.data["content_lang"] == "es"
```

Comment:
This does not prove any change happened. 
Hint: what was value before the call? Please update all tests.